package/yast-travis-ruby

```diff
@@ -186,9 +186,21 @@ function dump_settings() {
   echo
 }
 
+function set_rubocop_version() {
+  # here we need to set path exactly as update-alternatives needs, so find proper path from its output
+  RUBOCOP_PATH=$(update-alternatives --display rubocop | sed "/^[^[:space:]]\+$1"'/!d;s/^\([^[:space:]]\+\).*$/\1/')
+  update-alternatives --set rubocop "$RUBOCOP_PATH"
+}
+
 # run parallel rubocop if it is available,
 # otherwise fallback to standard rubocop call
 function run_rubocop() {
+  # set rubocop according to version that project use
+  if grep 'rubocop-0.71.0' .rubocop.yml; then
```

Comment:
NP: Maybe we should make this match more strict, this can accidentally match a comment like 

```ruby
# TODO: upgrade this old config to rubocop-0.71.0
```

e.g. at least ignore comments.